templates/lxd/index.html

```diff
@@ -4,337 +4,437 @@
+            <p>
+              Watch the <a href="https://ubuntu.com/engage/introduction-to-microcloud">“Introduction to MicroCloud”</a> webinar to learn more.
+            </p>
+          </div>
+          <div class="col-3 col-medium-2">
+            <h3 class="p-heading--5">Easily manage bare metal infrastructure with MAAS and LXD</h3>
+            <p>
+              MAAS can automatically deploy and configure LXD as part of the deployment of a physical machine. Or it can
+              be connected to an existing LXD deployment to dynamically create virtual machines on it.
+            </p>
+            <hr class="p-rule" />
+            <p>
+              <a href="https://maas.io/tutorials/build-a-maas-and-lxd-environment-in-30-minutes-with-multipass-on-ubuntu#1-overview">Access
               the tutorial about setting up your own MAAS and LXD environment&nbsp;&rsaquo;</a>
-          </p>
-        </div>
-        <div class="col-3 col-medium-2">
-          <h3 class="p-heading--5">Replace your VMware infrastructure with best-of-breed open source</h3>
-          <p>Open source brings cost savings, increased flexibility and higher quality for your infrastructure. Learn
-            how to build a fully functional cloud infrastructure on Ubuntu while ensuring feature parity with VMware
-            solutions.</p>
-          <hr class="p-rule">
-          <p>
-            <a href="https://ubuntu.com/engage/from-vmware-to-open-source">Learn more with the "From VMware to open
-              source" webinar&nbsp;&rsaquo;</a>
-          </p>
+            </p>
+          </div>
+          <div class="col-3 col-medium-2">
+            <h3 class="p-heading--5">Replace your VMware infrastructure with best-of-breed open source</h3>
+            <p>
+              Open source brings cost savings, increased flexibility and higher quality for your infrastructure. Learn
+              how to build a fully functional cloud infrastructure on Ubuntu while ensuring feature parity with VMware
+              solutions.
+            </p>
+            <hr class="p-rule" />
+            <p>
+              Read more about it in the <a href="https://ubuntu.com/engage/vmware-migration-scenarios">“VMware migration to Ubuntu-based infrastructure”</a> whitepaper.&nbsp;&rsaquo;
+            </p>
+          </div>
         </div>
       </div>
     </div>
-  </div>
-</section>
+  </section>
 
-<section class="p-section">
-  <div class="row--50-50">
-    <hr class="p-rule">
-    <div class="col">
-      <h3 class="p-heading--2">Learn more about LXD</h3>
-    </div>
-    <div class="col">
-      <div class="row--25-75">
-        <div class="col">
-          <h3 class="p-text--small-caps">Webinars</h3>
+  <section class="p-section">
+    <div class="row--50-50">
+      <hr class="p-rule" />
+      <div class="col">
+        <h3 class="p-heading--2">Learn more about LXD</h3>
+      </div>
+      <div class="col">
+        <div class="row--25-75">
```

Comment:
The shortcut classes (e.g row--50-50, row--25-75) should not be nested. See vanilla docs [here](https://vanillaframework.io/docs/patterns/grid#common-patterns). Perhaps you could try using a standard `row` class to break the columns up into your desired grids.